erpnext/healthcare/doctype/healthcare_practitioner/healthcare_practitioner.py

```diff
@@ -65,3 +66,23 @@ def on_trash(self):
 def validate_service_item(item, msg):
 	if frappe.db.get_value("Item", item, "is_stock_item") == 1:
 		frappe.throw(_(msg))
+
+def get_practitioner_list(doctype, txt, searchfield, start, page_len, filters=None):
+	fields = ["name", "first_name", "mobile_phone"]
+	match_conditions = build_match_conditions("Healthcare Practitioner")
+	match_conditions = "and {}".format(match_conditions) if match_conditions else ""
+
+	if filters:
+		filter_conditions = get_filters_cond(doctype, filters, [])
+		match_conditions += "{}".format(filter_conditions)
+
+	return frappe.db.sql("""select %s from `tabHealthcare Practitioner` where docstatus < 2
+		and (%s like %s or first_name like %s)
+		and active = 1
+		{match_conditions}
+		order by
+		case when name like %s then 0 else 1 end,
+		case when first_name like %s then 0 else 1 end,
+		name, first_name limit %s, %s""".format(match_conditions=match_conditions) %
+		(", ".join(fields), searchfield, "%s", "%s", "%s", "%s", "%s", "%s"),
+		("%%%s%%" % txt, "%%%s%%" % txt, "%%%s%%" % txt, "%%%s%%" % txt, start, page_len))
```

Comment:
this must be escaped or there will be injection. Also indentation!

Reply:
Thanks @rmehta for taking time to look into this. I have downloaded ESS-LLP branch and hopefully get time tonight to read up and learn about the correct way to avoid sql injection.

Reply:
Note that this is totally new code territory for me so if @pjamsheer is about please jump in.

Reply:
Ya I am trying to resolve.

Reply:
Thanks @rmetha, will make necessary changes